### ISSUE TITLE
Faulty link in the "Common WordPress Errors" HelpHub page

### ISSUE THREAD

**Issue by stevenlin-x** [WordPress/Documentation-Issue-Tracker#263]
## Issue Description
Under the section "[The White Screen of Death](https://wordpress.org/support/article/common-wordpress-errors/#the-white-screen-of-death)" of the [Common WordPress Errors](https://wordpress.org/support/article/common-wordpress-errors/) page, the sentence "The [WP_DEBUG feature](https://wordpress.org/support/article/editing-wp-config-php/#wp_debug) often provides additional information." has a faulty link

## URL of the Page with the Issue
https://wordpress.org/support/article/common-wordpress-errors/

## Section of Page with the issue
[The White Screen of Death](https://wordpress.org/support/article/common-wordpress-errors/#the-white-screen-of-death)

## Why is this a problem?
It should be linked to the proper link

## Suggested Fix
the underscore "_" in "#wp_debug" of the URL should be changed to hyphen "-"

In other words,

change the link
https://wordpress.org/support/article/editing-wp-config-php/#wp_debug
to
https://wordpress.org/support/article/editing-wp-config-php/#wp-debug

**Comment by stevenlin-x** [comment]
I've made the fix.
